Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -647,546 +552,351 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        /// <summary>
-        /// Tries to identify the string and return an object of that class.
-        /// </summary>
-        /// <param name="addr">String to parse.</param>
-        /// <param name="result">IPObject to return.</param>
-        /// <returns><c>true</c> if the value parsed successfully, <c>false</c> otherwise.</returns>
-        private static bool TryParse(string addr, out IPObject result)
+        /// <inheritdoc/>
+        public bool TryParseInterface(string intf, out List<IPData> result)
         {
-            if (!string.IsNullOrEmpty(addr))
+            result = new List<IPData>();
+            if (string.IsNullOrEmpty(intf))
             {
-                // Is it an IP address
-                if (IPNetAddress.TryParse(addr, out IPNetAddress nw))
-                {
-                    result = nw;
-                    return true;
-                }
+                return false;
+            }
 
-                if (IPHost.TryParse(addr, out IPHost h))
+            if (_interfaces != null)
+            {
+                // Match all interfaces starting with names starting with token
+                var matchedInterfaces = _interfaces.Where(s => s.Name.Equals(intf, StringComparison.OrdinalIgnoreCase)).OrderBy(x => x.Index).ToList();
+                if (matchedInterfaces.Any())
```

Comment:
It's probably the same under-the-hood, but don't we typically use `.Length > 0`?